alerts/plugindefinition.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   helmChart:
     name: alerts
     repository: oci://ghcr.io/cloudoperators/greenhouse-extensions/charts
-    version: 0.16.1
+    version: 0.17.0
```

Comment:
Please also bump the PluginDefinition version here - https://github.com/cloudoperators/greenhouse-extensions/blob/53aa9fc766274a8a70eb0d0cd9a17d39b247c164/alerts/plugindefinition.yaml#L9